### PR TITLE
Docs: More diagram fixes

### PR DESCRIPTION
**Pull request by JulietTeoh**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -165,6 +165,9 @@ An `Endpoint` contains the following attributes:
 * Before an API call is made, the `Response` object will be empty
 * Only when a Request#executeTimed(request) is called will a `Response` contain information about the API call response
 
+The following activity diagram summarizes what happens when a user executes a add command:
```
Comment (tjtanjin) [inline docs/DeveloperGuide.md:168]: Using `an add` might sound more correct here.

**Review by tjtanjin — APPROVED** [review]
LGTM, just 1 small nit to fix xD

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (ong6) [reply docs/DeveloperGuide.md:168]: The grind for comments is real

Comment (JulietTeoh) [reply docs/DeveloperGuide.md:168]: O man

**Pull request merged**